src/corelibs/U2Algorithm/src/util_msa_consensus/MSAConsensusUtils.cpp

```diff
@@ -67,14 +67,18 @@ QString MSAConsensusUtils::getConsensusPercentTip(const MultipleAlignment &ma, i
         return QString();
     }
     int gaps = 0;
-    for (int seq = 0; seq < nSeq; seq++) {
+    for (int seq = 0; seq < ma->getNumRows(); seq++) {
         uchar c = (uchar)ma->charAt(seq, pos);
         if (c >= 'A' && c <= 'Z') {
             int idx = c - 'A';
             freqs[idx].first++;
             freqs[idx].second = c;
         } else {
             // count gaps
+            if (ignoreLeadingTrailingGaps && ma->isGap(seq, pos, true)) {
```

Comment:
Add a method ma->isLeadingOrTrailingGap() and do not change ma->isGap()

Reply:
Note: there is not such thing as 'trailingGap' in the current gap model at all: isGap returns false for any position that can be considered as 'trailingGap' here.